services/src/main/scala/cromwell/services/metadata/MetadataQuery.scala

```diff
@@ -63,8 +63,10 @@ object MetadataValue {
       case _: Double | Float | _: java.lang.Double | _: java.lang.Float => new MetadataValue(value.toString, MetadataNumber)
       case _: Boolean | _: java.lang.Boolean => new MetadataValue(value.toString, MetadataBoolean)
       case other => new MetadataValue(other.toString, MetadataString)
-    }
+    } getOrElse Null
   }
+
+  val Null = new MetadataValue("", MetadataNull)
```

Comment:
what's the difference in behavior with these changes?

Reply:
The only difference in behavior is that if you pass `null` to the apply of `MetadataValue` it will create a `new MetadataValue("", MetadataNull)` which will translate to a `null` in the JSON, whereas before you'd get an empty string.

Reply:
Are the consumers of Cromwell metadata going to be okay with that change?

Reply:
Good question, I think it makes more sense this way but it'll probably be less painful to everyone to revert it in this case and pass a `MatadataValue.Null` explicitly for the execution event case.
Either way it will mean that the `endTime` of execution events can now be null which I don't think anyone will care about but it should definitely be made clear

Reply:
Do not inject a backwards incompatible change into the API plz

Reply:
Mokay I have a feeling this was a mistake 😄 I'll revert this whole thing